test/e2e_flags.go

```diff
@@ -63,7 +63,6 @@ func initializeServingFlags() *ServingEnvironmentFlags {
 	flag.BoolVar(&f.ResolvableDomain, "resolvabledomain", false,
 		"Set this flag to true if you have configured the `domainSuffix` on your Route controller to a domain that will resolve to your test cluster.")
 
-	flag.Parse()
```

Comment:
I suggest adding a comment to this function saying Parse() must not be called and linking to this PR (or the issue).

Reply:
Actually I think this should be replicated to all the other affected repos as well.

Reply:
Comment added. Will replicate to other affected repos once this is lgtm'ed